zsh-aws-vault.plugin.zsh

```diff
@@ -5,39 +5,40 @@ AWS_VAULT_PL_DEFAULT_PROFILE=${AWS_VAULT_PL_DEFAULT_PROFILE:-default}
 AWS_VAULT_PL_CHAR=${AWS_VAULT_PL_CHAR:-$'\u2601'} # "the cloud"
 AWS_VAULT_PL_BROWSER=${AWS_VAULT_PL_BROWSER:-''}
 AWS_VAULT_PL_MFA=${AWS_VAULT_PL_MFA:-''}
+AWS_VAULT_PL_BACKEND=${AWS_VAULT_PL_BACKEND:-'keychain'}
 
 #--------------------------------------------------------------------#
 # Aliases                                                            #
 #--------------------------------------------------------------------#
-alias av='aws-vault'
-alias avs='aws-vault server'
+alias av='aws-vault --backend=${AWS_VAULT_PL_BACKEND}'
+alias avs='aws-vault --backend=${AWS_VAULT_PL_BACKEND} server'
 
 #--------------------------------------------------------------------#
 # Convenience Functions                                              #
 #--------------------------------------------------------------------#
 function avl() {
-  aws-vault login $@
+  aws-vault --backend=${AWS_VAULT_PL_BACKEND} login $@
```

Comment:
I wonder, could you use the av alias you defined above to reduce needing to duplicate the --backend part throughout?

Reply:
That's a fantastic idea.  I'll update the PR.